<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignVCenter">
     <widget class="QWidget" name="header" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#header {
	background-color: #26272D;
	height: 45px;
	min-height: 45px;
	max-height: 45px;
	border-bottom: 1px solid black;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="headerContent">
         <property name="styleSheet">
          <string notr="true">QFrame#headerContent {
	padding: 0px;
	margin: 0px;
	margin-left: 12px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="logo">
            <property name="styleSheet">
             <string notr="true">image: url(:/images/resources/icons/monitor_alt.svg);
min-width: 20px;
min-height: 20px;
background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="title">
            <property name="styleSheet">
             <string notr="true">QLabel#title {
	font-family: &quot;Segoe UI&quot;, sans-serif;
	color:  white;
	font-size: 12px;
	font-weight: 400;
	padding-left: 8px;
	padding-top: 0px;
}</string>
            </property>
            <property name="text">
             <string>Intelligent Management System</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="windowOptions">
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
border: none;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnMinimize">
               <property name="styleSheet">
                <string notr="true">QPushButton#btnMinimize {
	image: url(:/images/resources/icons/minus_alt.svg);
	min-width: 18px;
	max-width: 18px;
	min-height: 45px;
	max-height: 45px;
	padding-left: 18px;
	padding-right: 18px;
	background-color: transparent;
	border: 0px;
}

QPushButton#btnMinimize:hover {
	background-color: #3d4145;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMaximize">
               <property name="styleSheet">
                <string notr="true">QPushButton#btnMaximize {
	image: url(:/images/resources/icons/square_alt.svg);
	min-width: 15px;
	max-width: 15px;
	padding-left: 20px;
	padding-right: 20px;
	min-height: 45px;
	max-height: 45px;
	background-color: transparent;
	border: 0px;
}

QPushButton#btnMaximize:hover {
	background-color: #3d4145;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnClose">
               <property name="styleSheet">
                <string notr="true">QPushButton#btnClose {
	image: url(:/images/resources/icons/x_alt.svg);
	min-width: 20px;
	max-width: 20px;
	min-height: 45px;
	max-height: 45px;
	padding-left: 18px;
	padding-right: 18px;
	background-color: transparent;
	border: 0px;
}

QPushButton#btnClose:hover {
	background-color: #e81123;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #181a20;



</string>
      </property>
      <widget class="QWidget" name="connectionPage">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-family: &quot;Segoe UI&quot;, sans-serif;
	font-size: 12px;
	font-weight: 400;
	color: white;
}

QPushButton {
	background-color: #5468ff;
	border-radius: 12px;
	
	min-height: 52px;
	max-height: 52px;
	
	font-family: &quot;Segoe UI&quot;, sans-serif;
	font-size: 14px;
	font-weight: 500;
	color: white;
}

QPushButton:hover {
	opacity: 0.6;
}

QLineEdit {
	background-color: #26272D;
	border-radius: 12px;
	padding-left: 20px;
	padding-right: 20px;
	
	min-height: 52px;
	max-height: 52px;
	
	font-family: &quot;Segoe UI&quot;, sans-serif;
	font-size: 14px;
	font-weight: 400;
	color: white;
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="connectionPageContent">
          <property name="styleSheet">
           <string notr="true">QFrame#connectionPageContent {
	padding: 30px;
	max-width: 480px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="txtTitle">
             <property name="styleSheet">
              <string notr="true">font-size: 24px;
font-weight: 600;
margin-bottom: 20px;</string>
             </property>
             <property name="text">
              <string>Connect to Server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtServerIP">
             <property name="styleSheet">
              <string notr="true">margin-bottom: 15px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Server IP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtServerPort">
             <property name="styleSheet">
              <string notr="true">margin-bottom: 20px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Server Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnConnect">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton#btnConnect {
}

QPushButton#btnConnect:hover {
	background-color: #4D5FE8;
}</string>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDisconnect">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton#btnDisconnect {
	background-color: #FF5454;
}

QPushButton#btnDisconnect:hover {
	background-color: #E04E4E;
}</string>
             </property>
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="txtReceivedData">
             <property name="styleSheet">
              <string notr="true">QPlainTextEdit#txtReceivedData {
	font-family: &quot;Segoe UI&quot;, sans-serif;
	font-size: 12px;
	font-weight: 400;
	color: white;
	border: 1px solid black;
	border-radius: 10px;
	padding: 10px;
	max-height: 100px;
	margin-top: 30px;
}</string>
             </property>
             <property name="plainText">
              <string>Connect...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
